Patch migrated from desktop gate, where it was listed as:
# date:2008-12-07 owner:wangke type:feature bugster:6788530
Patch2:                 libvisual-02-map.diff

which corresponds to:
Bug 15532087 - SUNBT6788530-SOLARIS_11 [nevada][nightly] gnome-settings-daemon
 does not start on login due to libvisual

Since this is for our build, probably not something we can get upstream.


--- libvisual-0.4.0/libvisual/map.remove_all	1970-01-01 08:00:00.000000000 +0800
+++ libvisual-0.4.0/libvisual/map.remove_all	2009-01-07 11:44:41.487159000 +0800
@@ -0,0 +1 @@
+hwcap_1 = OVERRIDE;
--- libvisual-0.4.0/map.remove_all	1970-01-01 08:00:00.000000000 +0800
+++ libvisual-0.4.0/map.remove_all	2009-01-19 14:00:20.369036000 +0800
@@ -0,0 +1 @@
+hwcap_1 = OVERRIDE;
